src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```diff
@@ -0,0 +1,158 @@
+@import "../../patternfly-imports";
```

Comment:
you can remove this

Reply:
should I be removing this on every component style sheet I'm working on?

Reply:
Yep, particularly for new components. If you rebase, you'll see they go away in existing components.